tests/test_config.py

```diff
@@ -153,6 +153,14 @@ def test_standalone_template(self):
 
         Config.create(template_file=base_path / "test.template.yaml")
 
+    def test_no_parameters(self):
+        base_path = "./" if os.getcwd().endswith("/tests") else "./tests/"
```

Comment:
Are we reasonably certain this is going to provide the correct value for `base_path` ?